packages/node_modules/@ciscospark/widget-meet/src/selector.js

```diff
@@ -12,13 +12,23 @@ const getOwnProps = (state, ownProps) => ownProps;
 const getCall = createSelector(
   [getOwnProps, getWidget, getCalls],
   (ownProps, widget, calls) => {
+    let call;
     if (ownProps.call) {
-      return ownProps.call;
+      ({call} = ownProps);
     }
-    if (calls.has(widget.callId)) {
-      return calls.get(widget.callId);
+    else if (calls.has(widget.callId)) {
+      call = calls.get(widget.callId);
     }
-    return widget.call;
+    else {
+      ({call} = widget);
+    }
+
+    // Patching in activeParticipantsCount with devices to support sip should be migrated to sdk
```

Comment:
To make that work correctly we'd actually need to alter the sdk to include devices as participants (currently it filters them out). Since a sip device can actually dial a space without being a user, the actual active participant count is not very accurate. Figure it was easier to patch it here, than to change the behavior of how the sdk works with devices.

Reply:
Understood. I wasn't asking about the sdk, the media module calculates the participants as well on every locus event, just wanted to know if this got overwritten as soon as an event comes in.

Reply:
Ahh, gotcha. Yea this happens outside of that, so we should be fine.